blocks/blog-home/blog-home.css

```diff
@@ -136,6 +143,19 @@ margin-inline-start: 20px;
   margin-inline-start: 10px;
 }
 
+@media (max-width: 600px){
```

Comment:
Let's only use min-width for consistency. You don't need to specify a breakpoint on these since they would be the default mobile style.